tests/tests/test-contract/test-contract-incr-loop.ts

```diff
@@ -1,29 +1,31 @@
 import { expect } from "chai";
-import { describeDevMoonbeam } from "../../util/setup-dev-tests";
+import { describeDevMoonbeam, describeDevMoonbeamAllEthTxTypes } from "../../util/setup-dev-tests";
 import { createContract, createContractExecution } from "../../util/transactions";
 
-describeDevMoonbeam("Contract loop creation", (context) => {
+describeDevMoonbeamAllEthTxTypes("Contract loop creation", (context) => {
   it("Should be initialized at 0", async () => {
-    const { contract, rawTx } = await createContract(context.web3, "TestContractIncr");
+    const { contract, rawTx } = await createContract(context, "TestContractIncr");
     await context.createBlock({ transactions: [rawTx] });
 
     expect(await contract.methods.count().call()).to.eq("0");
   });
 });
 
-describeDevMoonbeam("Contract loop increment", (context) => {
+describeDevMoonbeamAllEthTxTypes("Contract loop increment", (context) => {
   it("should increment contract state", async function () {
-    const { contract, rawTx } = await createContract(context.web3, "TestContractIncr");
+    const { contract, rawTx, contractAddress } = await createContract(context, "TestContractIncr");
     await context.createBlock({ transactions: [rawTx] });
-
     await context.createBlock({
       transactions: [
-        await createContractExecution(context.web3, {
+        await createContractExecution(context, {
           contract,
           contractCall: contract.methods.incr(),
         }),
       ],
     });
+    await context.createBlock();
+    await context.createBlock();
+    await context.createBlock();
```

Comment:
I don't think we need those anymore (I added them to test if it was the problem)